### DISCUSSION TITLE
SAdam implementation

### DISCUSSION THREAD

**Discussion by Ghilga** [tensorflow/compression#189] [Q&A]
Hi everyone,
Is there an implementation of the SAdam technique in this repository?
The paper `Efficient Nonlinear Transforms for Lossy Image Compression` mentioned that there is an implementation for it here, but I couldn't find it with the search tool.
I'd appreciate any help with this.

**Comment by jonaballe** [comment]
Hi Leonardo,

yes, Sadam is basically Adam applied to an RDFT parameterization of the
convolution kernels. This is implemented in the SignalConv2D class.

Hope this helps!
Johannes
